src/components/CharacterButton.vue

```diff
@@ -0,0 +1,329 @@
+                <q-menu
+                  no-parent-event
+                  anchor="top end"
+                  self="top start"
+                  transition-show="none"
+                  transition-hide="none"
+                  class="character-menu"
+                  v-model="subMenuOpenFlags[characterIndex]"
+                >
+                  <q-list>
+                    <q-item
+                      v-for="(style, styleIndex) in characterInfo.metas.styles"
+                      :key="styleIndex"
+                      clickable
+                      v-close-popup
+                      active-class="selected-character-item"
+                      :active="style.styleId === selectedVoice.styleId"
+                      @click="
+                        $emit('update:selectedVoice', {
+                          engineId: style.engineId,
+                          speakerId: characterInfo.metas.speakerUuid,
+                          styleId: style.styleId,
+                        })
+                      "
+                    >
+                      <q-avatar rounded size="2rem" class="q-mr-md">
+                        <q-img
+                          no-spinner
+                          no-transition
+                          :ratio="1"
+                          :src="characterInfo.metas.styles[styleIndex].iconPath"
+                        />
+                        <q-avatar
+                          rounded
+                          class="engine-icon"
+                          v-if="showEngineInfo"
+                        >
+                          <img
+                            :src="
+                              engineIcons[
+                                characterInfo.metas.styles[styleIndex].engineId
+                              ]
+                            "
+                          />
+                        </q-avatar>
+                      </q-avatar>
+                      <q-item-section v-if="style.styleName"
+                        >{{ characterInfo.metas.speakerName }} ({{
+                          style.styleName
+                        }})</q-item-section
+                      >
+                      <q-item-section v-else>{{
+                        characterInfo.metas.speakerName
+                      }}</q-item-section>
+                    </q-item>
+                  </q-list>
+                </q-menu>
+              </div>
+            </template>
+          </q-btn-group>
+        </q-item>
+      </q-list>
+    </q-menu>
+  </q-btn>
+</template>
+
+<script lang="ts">
+import { base64ImageToUri } from "@/helpers/imageHelper";
+import { useStore } from "@/store";
+import { CharacterInfo, Voice } from "@/type/preload";
+import { debounce } from "quasar";
+import { computed, defineComponent, PropType, ref } from "vue";
+
+export default defineComponent({
+  name: "CharacterButton",
+
+  props: {
+    characterInfos: { type: Array<CharacterInfo>, required: true },
```

Comment:
このファイルだけESLintが効いていなくて、原因がここでした。
配列の型はPropTypeで指定する必要があるみたいです。

これを修正するとESLintのエラーがいくつか検出されるようになるので、 `npm run fmt` の実行もお願いします。

```suggestion
    characterInfos: { type: Array as PropType<CharacterInfo[]>, required: true },
```

Reply:
ありがとうございます。
フォーマッターが上手く働いていないような気はしていましたがそんなところに原因があったとは…